src/main/java/com/sleekbyte/tailor/listeners/lengths/LengthListener.java

```diff
@@ -0,0 +1,98 @@
+package com.sleekbyte.tailor.listeners.lengths;
+
+import com.sleekbyte.tailor.antlr.SwiftBaseListener;
+import com.sleekbyte.tailor.antlr.SwiftParser;
+import com.sleekbyte.tailor.common.ConstructLengths;
+import com.sleekbyte.tailor.common.Location;
+import com.sleekbyte.tailor.common.Rules;
+import com.sleekbyte.tailor.output.Printer;
+import com.sleekbyte.tailor.utils.ListenerUtil;
+import org.antlr.v4.runtime.ParserRuleContext;
+
+import java.util.Set;
+
+/**
+ * Base class for length listeners.
+ */
+public abstract class LengthListener extends SwiftBaseListener {
+
+    protected ConstructLengths constructLengths;
+    protected Printer printer;
+    protected boolean traversedTreeForConstantDeclaration = false;
+    protected boolean traversedTreeForVarDeclaration = false;
+    protected Set<Rules> enabledRules;
+
+    public void setTraversedTreeForConstantDeclaration(boolean traversedTree) {
+        this.traversedTreeForConstantDeclaration = traversedTree;
+    }
+
+    public void setTraversedTreeForVarDeclaration(boolean traversedTree) {
+        this.traversedTreeForVarDeclaration = traversedTree;
+    }
+
+    @Override
```

Comment:
I originally created LengthListeber to make sure if a max was added, then it's corresponding min is also added.

Since it is decided that corresponding mins for all maxes will not be added/supported, I can remove LengthListener completely.